docs/kura-wires/script-components/nashorn-script-filter.md

```diff
@@ -95,4 +96,49 @@ record['myprop'] = newDoubleValue(123.456)
 
 // add the wire record to the output envelope
 output.add(record)
-```
\ No newline at end of file
+```
+
+#### Utility functions
+
+* `newWireRecord(void) -> WireRecordWrapper`: Creates a new mutable wire record. New properties can be added to the record as shown above. In order to emit the new record use the `output.add(WireRecordWrapper)` function.
+
+* `newByteArray(number) -> byte[]`: Creates a new mutable byte array of given length. The resulting array can be modified and the converted to a TypedValue using the `newByteArray(byte[]) -> TypedValue`.
+
+The following functions can be used for creating TypedValues. The resulting objects can be used for WireRecord property values. Some of these functions perform numeric conversions that could cause precision losses.
+
+* `newBooleanValue(boolean) -> TypedValue`: Creates a new TypedValue representing a boolean.
+
+* `newByteArrayValue(byte[]) -> TypedValue`: Creates a new TypedValue representing a byte array.
+
+* `newDoubleValue(number) -> TypedValue`: Creates a new TypedValue representing a double. The number provided as argument value will be coerced into a double.
+
+* `newFloatValue(number) -> TypedValue`: Creates a new TypedValue representing a float. The number provided as argument value will be coerced into a float.
+
+* `newIntegerValue(number) -> TypedValue`: Creates a new TypedValue representing an integer. The number provided as argument value will be coerced into an integer.
+
+* `newLongValue(number) -> TypedValue`: Creates a new TypedValue representing a long. The number provided as argument value will be coerced into a long.
+
+* `newStringValue(object) -> TypedValue`: Creates a new TypedValue representing a String. The object provided as argument value will be converted into a String using the `toString()` method.
+
+
+#### Logging
```

Comment:
Loggin, script context, and example script sections are common to both Nashorn and GraalVM components. Maybe it is better to add these in the "Introduction to the Script Components" page or even remove them since already described in the components metatypes.